src/jaegertracing/Span.h

```diff
@@ -21,6 +21,8 @@
 #include <memory>
 #include <mutex>
 
+#include "jaegertracing/Compilers.h"
```

Comment:
`Config.h` should have included this everywhere already.

Reply:
Why would I include Config.h in Span.h when I just need the compiler header fixing the MSVC compilation?

Reply:
I mean if `Compilers.h` is included in `Config.h`, then one of the includes below will bring `Compilers.h` here too. Please try to keep local imports at the end and sorted alphabetically. Don't rely on include ordering when you don't have to please.